docs/overview/learning-path-transition.md

```diff
@@ -0,0 +1,83 @@
+
+## Step 0: Prerequisites
+
+- Office Web Add-ins are essentially web applications embedded in Office. So, you should first have a basic understanding of web applications and how they are hosted on the web. There is an enormous amount of information about this on the Internet, in books, and in online courses. A good way to start if you have no prior knowledge of web applications at all is to search for "What is a web app?" on Bing.
+- The primary programming language you will use in creating Office Add-ins is JavaScript or TypeScript. You can think of TypeScript as a strongly-typed version of JavaScript. If you are not familiar with either of these languages, but you have experience with VBA, VB.Net, C#, you will probably find TypeScript easier to learn. Again, there is a wealth of information about these languages on the Internet, in books, and in online courses.
+
+## Step 1: Begin with fundamentals
+
+We know you're eager to start coding, but there are some things about Office Add-ins that you should read before you open your IDE or code editor.
+
+- [Office Add-ins Platform Overview](office-add-ins.md): Find out what Office Web Add-ins are and how they differ from older ways of extending Office, such as VSTO add-ins.
+- [Building Office Add-ins](office-add-ins-fundamentals.md): Get an overview of Office add-in development and lifecycle including tooling, creating an add-in UI, and using the JavaScript APIs to interact with the Office document.
+
+There are a lot of links in those articles, but if you're transitioning to Office Web Add-ins, we recommend that you come back here when you've read them and continue with the next section.
+
+## Step 2: Install tools and create your first add-in
+
+You've got the big picture now, so dive in with one of our quick starts. For purposes of learning the platform, we recommend the Excel quick start. There is a version that is based on Visual Studio and a version that is based in Node.js and Visual Studio Code. If you're transitioning from VSTO add-ins, you'll probably find the Visual Studio version the easiest to work with.
+
+- [Visual Studio](../quickstarts/excel-quickstart-jquery.md?tabs=visualstudio)
+- [Node.js and Visual Studio Code](../quickstarts/excel-quickstart-jquery.md?tabs=yeomangenerator)
+
+## Step 3: Code
+
+You can't learn to drive by reading the owner's manual, so start coding with this [Excel tutorial](../tutorials/excel-tutorial.md). You'll be using the Office JavaScript library and some XML in the add-in's manifest. There's no need to memorize anything, because you'll be getting more background about both in a later steps.
+
+## Step 4: Understand the JavaScript library
+
+First, get the big picture of the Office JavaScript library with this tutorial from Microsoft Learn: [Understand the Office JavaScript APIs](https://docs.microsoft.com/learn/modules/understand-office-javascript-apis/index).
```

Comment:
You can use relative paths to other docs. Also, the learning path with modules is now a module with units so updating link.
```suggestion
First, get the big picture of the Office JavaScript library with this tutorial from Microsoft Learn: [Understand the Office JavaScript APIs](/learn/modules/intro-office-add-ins/3-apis).
```